### PR TITLE
[DBC-211] Integrating tkey bridge with webview

### PR DESCRIPTION
**Pull request by vantam87**
## Description
- Implement torus service.
- Implement tkey bridge service.
- [document](https://docs.google.com/presentation/d/1Qupd-BMIBQXn02mEu3RmPuFgImCptQsMjq5Yu_RQ54w/edit#slide=id.gf9a74a8812_0_70)

## Note
- Upgrade to iOS 13 and update UIActivityIndicatorView as well because Torus SDK requires iOS 13.0 or later.

### CODE REVIEW

File: Wallet/Sources/Data/TKeyBridge/TKeyBridge.swift [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        let javascriptString = "saveTorusShare('\(postboxKey)','\(tsShare)','\(loginId)')"
+        webViewHandler.callJavascript(javascriptString)
+    }
+
+    private func reconstructKeyWithKeyShare(postboxKey: String, keyShare: String) {
+        let javascriptString = "reconstructKeyWithTorusShare('\(postboxKey)','\(keyShare)')"
+        webViewHandler.callJavascript(javascriptString)
+    }
+
+    private func generateRandom() -> String {
```
Comment (spica) [inline Wallet/Sources/Data/TKeyBridge/TKeyBridge.swift:111]: How about to change func name to 'generateRandomUUID()'?

Comment (vantam87) [reply Wallet/Sources/Data/TKeyBridge/TKeyBridge.swift:111]: This one generates a private key, I prefer to 'generateRandomPrivateKey' instead. What do you think?

File: Wallet/Sources/Supporting/Helpers/Constants/Constants+Torus.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//  Wallet
+//
+//  Created by Tam Nguyen on 27/10/2021.
+//  Copyright © 2021 MVL. All rights reserved.
+//
+
+extension Constants.Torus {
+
+    static let clientId: String = "354250895959-dneacv3fol73d6a6lf789mcjo2jjpbms.apps.googleusercontent.com"
+    static let verifierName: String = "clutch-google-testnet"
```
Comment (spica) [inline Wallet/Sources/Supporting/Helpers/Constants/Constants+Torus.swift:12]: This should be changed before app launching?

File: Wallet/Sources/Data/TKeyBridge/TKeyBridge.swift (hunk 1 above)
Comment (spica) [reply Wallet/Sources/Data/TKeyBridge/TKeyBridge.swift:111]: It's good to me

File: Wallet/Sources/Supporting/Helpers/Constants/Constants+Torus.swift (hunk 2 above)
Comment (vantam87) [reply Wallet/Sources/Supporting/Helpers/Constants/Constants+Torus.swift:12]: yup, we didn't have the environment for release yet. We'll update it then.

Comment (spica) [reply Wallet/Sources/Supporting/Helpers/Constants/Constants+Torus.swift:12]: OK.

**Review by spica — APPROVED** [review]

**Comment by spica** [comment]
Merge this after changed func name in comment

**Pull request merged**